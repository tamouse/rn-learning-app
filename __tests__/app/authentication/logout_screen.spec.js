import React from "react"
import renderer from "react-test-renderer"

import LogoutScreen from "../../../app/authentication/logout_screen";

describe('LogoutScreen', function () {

  it('renders the LogoutScreen', function () {
    const navigation = {}
    const screenProps = {
      loggedOut: jest.fn()
    }
    const wrapper = renderer.create(
      <LogoutScreen navigation={navigation} screenProps={screenProps}/>
    )
    expect(wrapper).toMatchSnapshot()
  });
});