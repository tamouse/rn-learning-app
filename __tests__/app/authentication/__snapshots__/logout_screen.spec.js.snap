// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutScreen renders the LogoutScreen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
      }
    }
  >
    Are you sure you want to log out?
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityLabel="logout button"
    accessibilityTraits="button"
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#007bff",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "opacity": 1,
        "paddingBottom": 5,
        "paddingLeft": 25,
        "paddingRight": 25,
        "paddingTop": 5,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#fafafa",
            "fontSize": 17,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          null,
        ]
      }
    >
      Confirm Log Out
    </Text>
  </View>
</View>
`;
